stories/assets/locale.ts

```diff
@@ -208,9 +208,9 @@ const locale: Record<string, string> = {
   turnOnRewardsTitle: 'Turn on Rewards',
   type: 'Type',
   uhOh: 'Uh oh…',
-  unVerifiedCheck: 'Check Again...',
-  unVerifiedPublisher: 'Not yet verified',
-  unVerifiedText: 'This creator has not yet signed up to receive contributions from Brave users.',
+  unVerifiedCheck: 'Refresh status',
```

Comment:
Keep in mind when you change the text values, it only changes for storybook and won't reflect the current string usage in `brave-core`

Reply:
yeah good spot. These should prob not be mixed in here but i cherry picked. i'll be sure to make them a part of the core sha pr